sidebars.js

```diff
@@ -1,17 +1,6 @@
 module.exports = {
   someSidebar: [
     'overview',
-    {
-      type: 'link',
-      label: 'gRPC API Reference',
-      href: 'https://buf.build/authzed/api/docs/main:authzed.api.v1',
-    },
-    {
-      type: 'link',
-      label: 'REST API Reference',
-      href: 'https://www.postman.com/authzed/workspace/spicedb/overview',
-    },
-    'support',
```

Comment:
These were all explicitly at the top due to their importance.

Reply:
I understood that. The main motivation to this change is that our docs are going to grow as we document APIs exclusive to SpiceDB Dedicated and SpiceDB Cloud. Folks will have to expand the `SpiceDB` menu to find them.

Admittedly I don't understand how this is going to affect the discoverability of these entries